bluetooth/src/androidMain/kotlin/com/mirego/trikot/bluetooth/android/AndroidAttributeProfileService.kt

```diff
@@ -16,15 +15,20 @@ class AndroidAttributeProfileService(
         HashMap<String, AndroidAttributeProfileCharacteristic>()
             as Map<String, AndroidAttributeProfileCharacteristic>
     ) { result, value ->
-        (mutableMapOf(
-            value.uuid.toString().toUpperCase(Locale.ROOT) to AndroidAttributeProfileCharacteristic(
-                value,
-                bluetoothDevice
-            )
-        ) + result).toMap()
+        (
+            mutableMapOf(
+                value.uuid.toString()
+                    .toUpperCase(Locale.ROOT) to
+                    AndroidAttributeProfileCharacteristic(
+                        value,
+                        bluetoothDevice
+                    )
+            ) + result
+            ).toMap()
```

Comment:
That's the proposed formatting from ktlint?

Reply:
Yes, this formatting is the result of `ktlint -F "src/**/*.kt"`